README.md

```diff
@@ -65,14 +65,20 @@ Run `stylelint` in a script like so:
 Since you're running `stylelint` natively, check out their documentation on how to run or extend it [here][link-stylelint-disable].
 
 ### Browser support
-Create an `.browserslistrc` file and extend the default recommended config like so:
-```js
+Create a `.browserslistrc` file and extend the default recommended config like so:
+```bash
 extends ./node_modules/@biotope/quality-gate/config/.browserslistrc
 # Add any other definition here. Example:
 # not ie 11
 ```
 
-Important: To make extends working, you have to start your linter which uses browserslist with dangerousExtend: true. For eslint and stylelint is this already pre-configured. More information [here][link-browserslist-extend].
+> Important: To make extends working, you have to start your linter which uses browserslist with dangerousExtend: true. For eslint and stylelint is this already pre-configured. More information [here][link-browserslist-extend].
+
+### Commitlint
+Create a `commitlint.config.js` file and extend the default recommended config like so:
```

Comment:
@tiagomapmarques can we please use .commitlintrc.js so it is more streanlined with other linters?